client/src/components/EditBox.js

```diff
@@ -27,21 +28,21 @@ export default function EditBox() {
     getLearningObj();
   }, [id]);
 
-  // const deleteLearningOb = (LearningID)=> {
-  //   console.log(LearningID)
-  //       fetch(`/api/learningobjectives/${LearningID}`, {
-  //           method: "DELETE",
-  //           headers: {
-  //             'Content-Type': 'application/json',
-  //           },
-  //         })
-  //           .then(res => res.json())
-  //           .then((data) =>data);
-
-  //       }
+  const deleteLearningOb = (LearningID)=> {
+    
+        fetch(`/api/learningobjectives/${LearningID}`, {
+            method: "DELETE",
+            headers: {
+              'Content-Type': 'application/json',
```

Comment:
I think you'll want to send the `token` header too, when you add the `authorization` middleware to the delete endpoint in the API